programmers/난이도별/level01.비밀지도/areumsheep.js

```diff
@@ -0,0 +1,14 @@
+function solution(n, arr1, arr2) {
+    var answer = [];
+    let str;
+    for(let i=0; i<n; i++){
+        str = Number(arr1[i] | arr2[i]).toString(2);
+        
+        let strLen = str.length;
+        if (strLen < n)
+            str = '0'.repeat(n - strLen) + str;
```

Comment:
```suggestion
        str = `${'0'.repeat(n)}{Number(arr1[i] | arr2[i]).toString(2)}`.substr(-n);
```
이런식으로 표현할 수 있습니다!

![image](https://user-images.githubusercontent.com/18749057/92775068-04ab2800-f3d9-11ea-959f-1473f22cf798.png)

Reply:
오 한 줄로 줄일 수 있군요... 감사합니다